app/src/main/java/eu/kanade/tachiyomi/util/chapter/ChapterSort.kt

```diff
@@ -28,10 +28,12 @@ class ChapterSort(val manga: Manga, val chapterFilter: ChapterFilter = Injekt.ge
     }
 
     fun <T : Chapter> getNextUnreadChapter(rawChapters: List<T>, andFiltered: Boolean = true,): T? {
+        var filteredChapters = chapterFilter.filterChaptersByScanlators(rawChapters, manga)
         val chapters = when {
             andFiltered -> chapterFilter.filterChapters(rawChapters, manga)
-            else -> rawChapters
+            else -> filteredChapters
         }
```

Comment:
since chapterFilter.filterChapters already does filtering
```kotlin
        val chapters = when {
            andFiltered -> chapterFilter.filterChapters(rawChapters, manga)
            else ->  chapterFilter.filterChaptersByScanlators(rawChapters, manga)
        }
```

Reply:
Missed this one

Reply:
thats weird, my local shows that its changed and i couldn't get it recognize its different so i changed it on github... I think that worked